learning_resources/views.py

```diff
@@ -25,6 +26,24 @@ class DefaultPagination(LimitOffsetPagination):
     max_limit = 100
 
 
+@extend_schema_view(
+    list=extend_schema(
+        summary="List",
+        description="Get a paginated list of learning resources.",
+    ),
+    retrieve=extend_schema(
+        summary="Retrieve",
+        description="Retrieve a single learning resource.",
+    ),
+    new=extend_schema(
+        summary="List New",
+        description="Get a paginated list of newly released resources.",
+    ),
+    upcoming=extend_schema(
+        summary="List Upcoming",
+        description="Get a paginated list of upcoming resources.",
+    ),
+)
```

Comment:
This just makes the generated docs a little bit nicer.

We do not need this, and if it's annoying, we can get rid of it until we decide to do something with the docs.

Without it:
- documentation descriptions come from docstrings
- I'm not sure where the summaries come from (which are used for left sidebar links in http://localhost:8063/api/v1/schema/redoc)